lib/assets/javascripts/views/devise/devise.js

```diff
@@ -0,0 +1,40 @@
+import ariatiseForm from '../../utils/ariatiseForm';
+import { addMatchingPasswordValidator, addShowHidePasswordControl } from '../../utils/passwordHelper';
+import { SHOW_SELECT_ORG_MESSAGE, SHOW_OTHER_ORG_MESSAGE } as constants from '../constants';
```

Comment:
the syntax for this import is not fine. Remove as constants and the add the URL as ''../../constants please